packages/server/lib/server-base.ts

```diff
@@ -148,6 +150,13 @@ export abstract class ServerBase<TSocket extends SocketE2E | SocketCt> {
     })
 
     this.requestedWithAndCredentialManager = requestedWithAndCredentialManager
+    const perfObserver = new PerformanceObserver((items) => {
```

Comment:
I think we can ditch this now?

Reply:
i think everything in this file could be removed